docs/install-server.md

```diff
@@ -46,9 +51,14 @@ Examples:
                     <assemblyArtifact>
                         <groupId>com.ibm.websphere.appserver.runtime</groupId>
                         <artifactId>wlp-webProfile7</artifactId>
-                        <version>8.5.5.7</version>
+                        <version>17.0.0.2</version>
```

Comment:
Should we use 17.0.0.2 before it is released?  The estimate date has been published, but my concern is that if people copy/paste this while using the snapshot before June 23, they will get errors.